src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAccountTransferAllowance.sql

```diff
@@ -4,12 +4,13 @@
 AS
 	SET NOCOUNT ON
 
-	SELECT	SUM(Amount) * @allowancePercentage 
+	SELECT	(SUM(lines.Amount) * @allowancePercentage) - SUM(transfers.Amount)
```

Comment:
I don't think the sums will work here with the inner join on account ID, the number of records you'll get back is effectively O(N<sup>2</sup>). Unless I'm missing something?

Reply:
this has now been changed.